Tools/plot_parallel.py

```diff
@@ -135,15 +143,12 @@ def plot_snapshot(filename):
     if plotlib == 'matplotlib':
         plt.xlabel('z (m)')
         plt.ylabel('x (m)')
-        plt.title(args.field + ' at iteration ' + str(iteration) +
-                  ', time = ' + str(ds.current_time))
-        plt.savefig(args.path + '/plt_' + args.field + '_' + plotlib + '_' +
-                    str(iteration).zfill(5) + '.png', bbox_inches='tight', dpi=300)
+        plt.title('%s at iteration %d, time = %e s'%(args.field, iteration, ds.current_time))
+        plt.savefig('plt_%s_%s_%05d.png'%(args.field, plotlib, iteration), bbox_inches='tight', dpi=300)
```

Comment:
Did you remove `args.path + ` on purpose? As far as I am concerned, I like when the results are saved close to the data. Alternatively, do you think it would be worth adding an option
```py
parser.add_argument('--path_img', default='.',
	                    help='path of directory where images are saved.')
```
the default can be `.` (new version) or `drags/plotfiles` (old version).

Reply:
I can add the args.path prefix back in. My preference is that the images appear in the directory where this is used, but putting them with the plotfiles is fine. I can add the optional path, though I would call it "image_dir".

Reply:
Ok, perfect.